modules/browsemap/src/main/java/com/sygic/maps/module/browsemap/extensions/BrowseMapExtensions.kt

```diff
@@ -24,57 +24,55 @@
 
 package com.sygic.maps.module.browsemap.extensions
 
-import android.annotation.SuppressLint
 import android.app.Application
 import com.sygic.maps.module.browsemap.R
 import com.sygic.maps.module.common.component.*
 
-@SuppressLint("Recycle")
 fun MapFragmentInitComponent.resolveAttributes(app: Application) {
-    app.obtainStyledAttributes(attributes, R.styleable.BrowseMapFragment)?.let { typedArray ->
```

Comment:
teoreticky ak tento file revertnes, vyhneme sa konfliku (realne to je len kozmetika ak dobre vidim, tak funkcnost to neovplyvnuje)